src/ensemble_scheduler/ensemble_scheduler.cc

```diff
@@ -1385,6 +1483,13 @@ EnsembleScheduler::EnsembleScheduler(
   // This config field is filled internally for ensemble models
   info_->is_decoupled_ = config.model_transaction_policy().decoupled();
 
+  // field to check if response cache enabled in the ensemble model config.
+  info_->is_cache_enabled_ =
+      config.response_cache().enable() && is_->ResponseCacheEnabled();
+  if (info_->is_cache_enabled_) {
+    LOG_VERBOSE(1) << "Top Level Ensemble Request Caching Enabled";
+  }
```

Comment:
This can be removed, as there may be multiple ensemble models, and can be queried from ensemble model config if needed

Reply:
this one was resolved but wasn't changed